examples/job_attachments/README.md

```diff
@@ -0,0 +1,198 @@
+# Deadline Cloud Job Attachments CLI "Beta"
+
+## Purpose
+
+Deadline Cloud offers the [Job Attachments](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/storage-job-attachments.html) feature to transfer files back and forth between your workstation and AWS Deadline Cloud. For an indepth look into the Job Attachments feature, please refer to the Job Attachments [developer guide](https://docs.aws.amazon.com/deadline-cloud/latest/developerguide/submitting-files-with-a-job.html).
+
+Where does Job Attachments come into use in the Deadline Cloud job life cycle? 1) Job Submission, 2) Moving data to workers for job execution, 3) Copying job execution output back to the Job Attachments bucket and 4) Downloading Job Outputs.
+
+This README outlines CLIs and Python APIs to interface with each step in the Job Attachment process.
+
+Concepts:
+- Content Addressable Storage (CAS) - [wiki](https://en.wikipedia.org/wiki/Content-addressable_storage)
+
+## TLDR; Quick start
+
+### Creating a Job Attachments manifest "snapshot" to capture files for upload:
+```
+deadline manifest snapshot \
+    --root ./path/to/files \
+    --destination ./path/for/snapshot/manifest/ \
+    --name data-snapshot \
+    --include "*.txt" \                               (Optional)
+    --exclude "*.tmp"                                 (Optional)
+```
+
+### Uploading a manifest file to S3, useful for subsequent job submissions.
+```
+deadline manifest upload \
+    --farm-id farm-12345678123456781234567812345678 \
+    --queue-id queue-98765432987654329876543298765432 \
+    --s3-manifest-prefix where/to/store/manifests \                         (Optional)
+    --json \                                                                (Optional)
+    --profile \                                                             (Optional)
+    /path/for/snapshot/manifest/data-snapshot-2024-11-15T15-46-40.manifest
+```
+
+### Uploading files captured by Job Attachment "snapshot" to S3:
+```
+deadline attachment upload \
```

Comment:
One of `--root-dirs` or `--path-mapping-rules` is required, but not both. They're used for determining where the attachments are for upload.

Reply:
Right! I need to add that.